services/softlayer_account_test.go

```diff
@@ -96,6 +96,19 @@ var _ = Describe("SoftLayer_Account_Service", func() {
 		})
 	})
 
+	Context("#GetIscsiNetworkStorageWithFilter", func() {
+		BeforeEach(func() {
+			fakeClient.DoRawHttpRequestResponse, err = common.ReadJsonTestFixtures("services", "SoftLayer_Account_Service_getNetworkStorage.json")
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("returns an array of datatypes.SoftLayer_Network_Storage", func() {
+			iscsiNetworkStorage, err := accountService.GetIscsiNetworkStorageWithFilter("fake-filter")
```

Comment:
I usually like the use of `fake-xyz` but I wonder if this correct thing here.

Would it make sense to check the filters? That is have code that will not accept certain filters? Are the list of possible filters known?